pkg/action/get_metadata.go

```diff
@@ -57,13 +65,30 @@ func (g *GetMetadata) Run(name string) (*Metadata, error) {
 	}
 
 	return &Metadata{
-		Name:       rel.Name,
-		Chart:      rel.Chart.Metadata.Name,
-		Version:    rel.Chart.Metadata.Version,
-		AppVersion: rel.Chart.Metadata.AppVersion,
-		Namespace:  rel.Namespace,
-		Revision:   rel.Version,
-		Status:     rel.Info.Status.String(),
-		DeployedAt: rel.Info.LastDeployed.Format(time.RFC3339),
+		Name:         rel.Name,
+		Chart:        rel.Chart.Metadata.Name,
+		Version:      rel.Chart.Metadata.Version,
+		AppVersion:   rel.Chart.Metadata.AppVersion,
+		Dependencies: rel.Chart.Metadata.Dependencies,
+		Annotations:  rel.Chart.Metadata.Annotations,
+		Namespace:    rel.Namespace,
+		Revision:     rel.Version,
+		Status:       rel.Info.Status.String(),
+		DeployedAt:   rel.Info.LastDeployed.Format(time.RFC3339),
 	}, nil
 }
+
+// FormattedDepNames formats metadata.dependencies names into a comma-separated list.
+func (m *Metadata) FormattedDepNames() string {
+	if len(m.Dependencies) == 0 {
+		return "<none>"
```

Comment:
To align with other fields that do not produce a result (a good example is App Version), recommend following the same pattern and omitting any output

Reply:
Done. I've left the omitempty in for the json and yaml outputs. Would you rather that I let them displace `null`?

Reply:
I think he meant that actually omit annotations if there are none.

Reply:
In table view, for an empty values, appVersion shows `APP_VERSION: `. I've gone with the same output here for ANNOTATIONS and DEPENDENCIES.

In YAML/JSON view appVersion displays the empty output explicitly `appVersion: ""`. I've gone with omitempty in these cases, viz. the annotationa and the dependencies fields would be missing if they are null.